src/main/java/Lottery.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
```

Comment:
Random and ColorSupplier should be class field, don't forget about appropriate access modifier
100 should be class constant, don't forget about CONSTANT_CASE